bip-0003.md

```diff
@@ -657,10 +664,19 @@ feedback, and helpful comments.
     aforementioned can be represented by _Closed_ without significantly impacting the information quality of the
     overview table. Where the many Status variants provided minuscule additional information, the simplification is more
     valuable and the Changelog section now collects specific details.
-[^acceptance]: **Why does the BIPs repository no longer track adoption?**  
-    BIP 2 made an attempt to gather community feedback into comment summaries in BIPs directly. Given the low adoption
-    and corresponding low information quality of the summaries that resulted from that feature, this BIP instead intends
-    to leave the evaluation of BIPs to the audience.
+[^acceptance]: **What is meant by acceptance?**  
```

Comment:
Per my comment above, perhaps "adoption" would be a better term, as "acceptance" could be construed as acceptance into the BIPs repository (even if out of scope).

Reply:
Switched to using "Publication" and "Adoption" as described above. Also slightly amended this footnote.